### PR TITLE
Upgrade Jellyfin-Server-Windows to .Net Core 3.1

### PR DESCRIPTION
**Pull request by pusta**
This PR addresses the .Net Core version that the Windows Jellyfin try is built against.  The current version, .Net Core 3.0, went End of Life  on March 2nd, 2020.  With this change, the Jellyfin Windows Tray application is now built against .Net Core 3.1, which is an LTS release.  Along with this change, the ContentMenu, which was deprecated in .Net Core 3.1,  was updated to ContextMenuStrip.  The MenuItems, also deprecated in this release, were also updated to ToolStripMenuItem.  All other functionality was tested, and left the same

### CODE REVIEW

File: Jellyfin.Windows.Tray/Properties/Resources.resx [hunk 1]
```diff
@@ -119,6 +119,6 @@
   </resheader>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="JellyfinIcon" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\JellyfinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+    <value>..\Resources\JellyFinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```
Comment (nielsvanvelzen) [inline Jellyfin.Windows.Tray/Properties/Resources.resx:122]:
```suggestion
    <value>..\Resources\JellyfinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

The `f` should be lowercase

Comment (pusta) [reply Jellyfin.Windows.Tray/Properties/Resources.resx:122]: I caught that before I submitted.  I just checked and my branch does have the lower case "f"

Comment (nielsvanvelzen) [reply Jellyfin.Windows.Tray/Properties/Resources.resx:122]: I think something went wrong as the file itself is still named "JellyFinIcon.ico" and the resx file still points to that name.

[This Stack Overflow question might help](https://stackoverflow.com/a/24979063)

Comment (pusta) [reply Jellyfin.Windows.Tray/Properties/Resources.resx]: OK, I think I caught all the capital "F" instances in my latest commit.  Sorry about that.

Comment (nielsvanvelzen) [reply Jellyfin.Windows.Tray/Properties/Resources.resx]: Unfortunately the file still got the wrong name, see [here](https://github.com/jellyfin/jellyfin-server-windows/pull/26/files#diff-83163a142865202d1c729e3ef871cd4c)

Comment (pusta) [reply Jellyfin.Windows.Tray/Properties/Resources.resx]: Maybe third times the charm?  I renamed it in Git and pushed.

Comment (nielsvanvelzen) [reply Jellyfin.Windows.Tray/Properties/Resources.resx]: Yup, the file naming looks good now

Comment (pusta) [reply Jellyfin.Windows.Tray/Properties/Resources.resx]: Thank you for the help!!

**Pull request merged**